security-advisories/msmq-permissions.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Security Advisory 2017-01-03
+summary: NServiceBus MSMQ permissions vulnerability
+reviewed: 2017-01-03
+tags:
+ - MSMQ
+ - Transport
+ - Security
+---
+
+This advisory discloses a security vulnerability that has been found in NServiceBus and fixed in a recently released hotfix.
+* All endpoints should be upgraded to the latest version of the NServiceBus package to fix this vulnerability if:
+  * using the NServiceBus MSMQ transport,
+  * running the NServiceBus [installers](/nservicebus/operations/installers.md) to provision endpoints,
+  * customized the queue permissions by revoking access for Anonymous and/or Everyone account(s) .  
```

Comment:
This and line 12 does not read well. I think it should be something like 

>**user** customized the queue ...

or

> the queue permissions have been customized ...